config/locales/en.yml

```diff
@@ -868,36 +1595,108 @@ en:
       subscribe: Subscribe
       subscribed_on_another_device: Subscribe on this Device
       unsubscribe: Unsubscribe
+  controllers:
+    ApplicantsController:
+      link_expired: That one-time link has expired, or is invalid. If you have already completed enrollment, please sign in.
+      welcome: Welcome to %{school_name}!
+    CoursesController:
+      errors: "There were errors with your submission: %{form_errors}"
+      sent_mail: We've sent you a verification mail. It should reach you in less than a minute. Click the link in the email to sign up, and get started.
+    Schools__CoursesController:
+      teams_active: Teams marked active successfully!
+    UsersController:
+      link_expired: That link has expired or is invalid. Please try again.
+    Users__OmniauthCallbacksController:
+      add_facebook: Please <a href="https://www.facebook.com/settings?tab=applications" target="_blank" rel="noopener">remove \'Pupilfirst\' from your authorized apps list</a> and try signing in again.
+      add_github: Please <a href="https://github.com/settings/profile" target="_blank" rel="noopener">add a public email address to your Github profile</a> and try again.
+      add_other: Please sign in using another method.
+      denied: Authentication was denied. Please try again.
+      denied_by: Authentication was denied by %{provider}. Please try again.
+      email_unregistered: "Your email address: %{email} is unregistered."
+      not_receive_email: "We're sorry, but we did not receive your email address from %{provider_name}. "
```

Comment:
@prashanth726 Could we not repeat _controllers_ in the key names here?

What I mean is, right now, it's `controllers.ApplicantsController.key`.

Instead, could we have `controllers.applicants.key`, or even better `controllers.applicants.ACTION.key`?